diff -urN highlight-3.34.orig/src/makefile highlight-3.34/src/makefile
--- highlight-3.34.orig/src/makefile	2017-01-24 22:56:25.702768453 +0100
+++ highlight-3.34/src/makefile	2017-01-24 22:57:44.332773964 +0100
@@ -2,7 +2,7 @@
 # This file will compile the highlight library and binaries.
 # See INSTALL for instructions.
 
-# Add -DHL_DATA_DIR=\"/your/path/\" to CFLAGS if you want to define a
+# Add -DHL_DATA_DIR=\"/your/path/\" to CXXFLAGS if you want to define a
 # custom installation directory not listed in INSTALL.
 # Copy *.conf, ./langDefs, ./themes and ./plugins to /your/path/.
 # See ../makefile for the definition of ${data_dir}
@@ -19,13 +19,13 @@
 
 QMAKE ?= qmake
 
-CFLAGS:=-Wall -O2 ${CFLAGS} -DNDEBUG -std=c++11 -D_FILE_OFFSET_BITS=64
+CXXFLAGS:=-Wall -O2 ${CFLAGS} -DNDEBUG -std=c++11 -D_FILE_OFFSET_BITS=64
 
-#CFLAGS:= -fPIC -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -fasynchronous-unwind-tables -fstack-clash-protection
+#CXXFLAGS:= -fPIC -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -fasynchronous-unwind-tables -fstack-clash-protection
 
-#CFLAGS:=-ggdb -O0 ${CFLAGS} -std=c++11
+#CXXFLAGS:=-ggdb -O0 ${CFLAGS} -std=c++11
 
-CFLAGS_DILU=-fno-strict-aliasing
+CXXFLAGS_DILU=-fno-strict-aliasing
 
 SO_VERSION=4.0
 
@@ -63,7 +63,7 @@
 endif
 
 ifdef PIC
-	CFLAGS+=-fPIC
+	CXXFLAGS+=-fPIC
 endif
 
 ifneq ($(OS), Windows_NT)
@@ -73,7 +73,7 @@
 #LDFLAGS = ${LDFLAGS} -s
 #LDFLAGS= -Wl,--as-needed
 
-CXX_COMPILE=${CXX} ${CFLAGS} -c -I ${INCLUDE_DIR} ${LUA_CFLAGS}
+CXX_COMPILE=${CXX} ${CXXFLAGS} -c -I ${INCLUDE_DIR} ${LUA_CFLAGS}
 
 # Data directories (data dir, configuration file dir)
 CXX_DIR=-DHL_DATA_DIR=\"${HL_DATA_DIR}\" -DHL_CONFIG_DIR=\"${HL_CONFIG_DIR}\"
@@ -235,7 +235,7 @@
 LuaUtils.o: ${DILU_DIR}LuaUtils.cpp
 	${CXX_COMPILE} ${DILU_DIR}LuaUtils.cpp
 LuaValue.o: ${DILU_DIR}LuaValue.cpp
-	${CXX_COMPILE} ${CFLAGS_DILU} ${DILU_DIR}LuaValue.cpp
+	${CXX_COMPILE} ${CXXFLAGS_DILU} ${DILU_DIR}LuaValue.cpp
 LuaVariable.o: ${DILU_DIR}LuaVariable.cpp
 	${CXX_COMPILE} ${DILU_DIR}LuaVariable.cpp
 LuaWrappers.o: ${DILU_DIR}LuaWrappers.cpp
