--- highlight-2.9/src/makefile.orig	2009-04-23 05:32:21.000000000 +0200
+++ highlight-2.9/src/makefile	2009-05-27 18:57:01.000000000 +0200
@@ -2,7 +2,7 @@
 # This file will compile the highlight library and binaries.
 # See INSTALL for instructions.
 
-# Add -DHL_DATA_DIR=\"/your/path/\" to CFLAGS if you want to define a
+# Add -DHL_DATA_DIR=\"/your/path/\" to CXXFLAGS if you want to define a
 # custom installation directory not listed in INSTALL.
 # Copy *.conf, ./langDefs, ./themes amd ./indentSchemes to /your/path/.
 # See ../makefile for the definition of ${data_dir}
@@ -17,8 +17,8 @@
 
 CXX=c++
 
-CFLAGS:=-O2 ${CFLAGS}
-#CFLAGS:=-ggdb ${CFLAGS}
+CXXFLAGS:=-O2 ${CXXFLAGS}
+#CXXFLAGS:=-ggdb ${CXXFLAGS}
 
 SO_VERSION=2.13
 
@@ -38,7 +38,7 @@
 	HL_DATA_DIR = /usr/share/highlight/
 endif
 ifdef PIC
-	CFLAGS+=-fPIC
+	CXXFLAGS+=-fPIC
 endif
 
 # Do not strip by default (Mac OS X lazy pointer issues)
@@ -48,7 +48,7 @@
 #LDFLAGS = ${LDFLAGS} -s
 #LDFLAGS= -Wl,--as-needed
 
-CXX_COMPILE=${CXX} ${CFLAGS} -c -I ${CORE_DIR}
+CXX_COMPILE=${CXX} ${CXXFLAGS} -c -I ${CORE_DIR}
 
 # Data directories (data dir, configuration file dir)
 CXX_DIR=-DHL_DATA_DIR=\"${HL_DATA_DIR}\" -DHL_CONFIG_DIR=\"${HL_CONFIG_DIR}\"
